lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.R

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env Rscript
+
+library(plyr, quietly=TRUE)
+library(reshape, warn.conflicts=FALSE, quiet=TRUE)
+library(ggplot2, quietly=TRUE)
+library(getopt)
+
+opt <- getopt(
```

Comment:
I feel like I would put all the option parsing and default into its own function.